website/server/controllers/api-v3/user.js

```diff
@@ -1326,4 +1326,33 @@ api.setCustomDayStart = {
   },
 };
 
+/**
+* @api {post} /user/notifications/clear/:invitedUsername/:groupId Clear a notification of an accepted invite
```

Comment:
Our convention for these `@api` lines is that the path starts with `/api/v3/`  (see the previous route in this file, for example). Also, @khipkin's comment below is correct - there's a mismatch between this comment and the `url` in the code.